test/integration/BUILD

```diff
@@ -98,13 +100,16 @@ kt_jvm_library(
     ],
 )
 
-kt_jvm_test(
+typedb_kt_test(
     name = "test-quickstart",
     srcs = ["QuickstartTest.kt"],
     data = ["//test/data:query-string-files"],
     test_class = "com.vaticle.typedb.studio.test.integration.QuickstartTest",
     runtime_deps = skiko_runtime_platform,
-    deps = compose_test_libraries
+    deps = compose_test_libraries,
+    server_mac_artifact = "@vaticle_typedb_artifact_mac//file",
+    server_linux_artifact = "@vaticle_typedb_artifact_linux//file",
+    server_windows_artifact = "@vaticle_typedb_artifact_windows//file",
```

Comment:
There's a bit of repetition here, but in typedb-client-java we provide the artifacts explicitly every time, see:
https://github.com/vaticle/typedb-client-java/blob/master/test/integration/BUILD
https://github.com/vaticle/typedb-client-java/blob/master/test/behaviour/concept/thing/entity/BUILD

Reply:
I think the client-java targets may not themselves be ideal yet.

However, it's difficult to improve much on what we have already, here. We already tried creating a macro, and that resulted in the definition of `deps` becoming awkward. So let's stick with the current approach.